### PR TITLE
Feat/RTENU-216 map area list alfresco folder

### PR DESCRIPTION
**Pull request by NinaDang97**

### CODE REVIEW

File: packages/frontend/src/components/Map/types.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Feature } from 'geojson';
+
+export interface FinlandMapFeature extends Feature {
+  properties: {
+    id: number;
+    kpalue: number;
+    tilirataos: number;
```
Comment (mehtis) [inline packages/frontend/src/components/Map/types.ts:7]: As previously, these should be translated. Though I don't quite know what the translations would be. Any idea @saneDG? 🤔

Comment (saneDG) [reply packages/frontend/src/components/Map/types.ts:7]: These are tricky.. 
kpalue -> maintenanceArea
tilirataos(tilirataosa??) -> ???

Comment (NinaDang97) [reply packages/frontend/src/components/Map/types.ts:7]: This is a bit tricky, Finnish keys represent original Json data

Comment (mehtis) [reply packages/frontend/src/components/Map/types.ts:7]: Right. In that case I'd say this is fine then.

**Pull request merged**